src/translations/en.json

```diff
@@ -23,10 +22,9 @@
     "deleteStudyError": "You are not the owner of this study",
     "deleteElementsFailure": "Problem encountered while deleting {stn, plural, one {an element} other {elements}}. Not deleted : {problematic}",
     "moveElementsFailure": "Problem encountered while moving {stn, plural, one {an element} other {elements}}. Not moved : {problematic}",
-    "caseExist": "Does the case already exist?",
-    "studyName": "study name",
+    "studyName": "name",
     "studyNameProperty": "Study name: ",
-    "studyDescription": "study description",
+    "studyDescription": "description (optional)",
```

Comment:
then the key should be "DescriptionOpt" and could be used everywhere without duplication ?

Reply:
fixed